README.md

```diff
@@ -21,3 +21,11 @@ docker build -t mock-avs-health-checker:latest https://github.com/NethermindEth/
 ```
 
 This command generates the docker image `mock-avs-health-checker:latest` which can be used to run the service.
+
+### Build `plugin` image
+
+```bash
+docker build -t mock-avs-plugin:latest https://github.com/NethermidEth/mock-avs-src.git#main:plugin
+```
+
+This command generate the docker image `mock-avs-plugin:latest` which can be used to run the plugin.
```

Comment:
```suggestion
This command generates the docker image `mock-avs-plugin:latest`, which can be used to run the plugin.
```

Reply:
Done https://github.com/NethermindEth/mock-avs-src/pull/1/commits/b60e88d958b13586f36d0f25997a607f4cc1103f

Reply:
You forgot the comma. I'm approving anyways